src/components/TVOptionItem.css

```diff
@@ -0,0 +1,9 @@
+.ListIcon {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+}
+imgIcon{
```

Comment:
Volgens de aandachtspuntenlijst is het beter om hier kleine letters te gebruiken.